client/packages/system/src/Encounter/ListView/Toolbar.tsx

```diff
@@ -35,7 +35,7 @@ export const Toolbar: FC<{ filter: FilterController }> = () => {
             },
             {
               type: 'group',
-              name: 'Date/Time Range',
+              name: 'label.start-datetime',
```

Comment:
The translation key is missing! More importantly, there's no call to `t` to translate.
Perhaps just use `label.date`? Then it matches the column header: `name: t('label.date'),`

<img width="247" alt="image" src="https://github.com/msupply-foundation/open-msupply/assets/9192912/e8933c4d-b257-43e7-bb53-ee9039cc176a">

Reply:
Needs to call `t` to translate 🤔.... also date doesn't tell you if it's the start date time or end date time until you click on it since encounter has both...